configuration/configuration.py

```diff
@@ -187,6 +187,14 @@ def read_secret(secret_name):
 # prefer IPv4 instead.
 PREFER_IPV4 = os.environ.get('PREFER_IPV4', 'False').lower() == 'true'
 
+# Remote authentication support
+REMOTE_AUTH_ENABLED = os.environ.get('REMOTE_AUTH_ENABLED', 'False').lower() == 'true'
+REMOTE_AUTH_BACKEND = os.environ.get('REMOTE_AUTH_BACKEND', 'netbox.authentication.RemoteUserBackend')
+REMOTE_AUTH_HEADER = os.environ.get('REMOTE_AUTH_HEADER', 'HTTP_REMOTE_USER')
+REMOTE_AUTH_AUTO_CREATE_USER = os.environ.get('REMOTE_AUTH_AUTO_CREATE_USER', 'True').lower() == 'true'
+REMOTE_AUTH_DEFAULT_GROUPS = list(filter(None, os.environ.get('REMOTE_AUTH_DEFAULT_GROUPS', '').split(' ')))
+REMOTE_AUTH_DEFAULT_PERMISSIONS = ast.literal_eval(os.environ.get('REMOTE_AUTH_DEFAULT_PERMISSIONS', '{}'))
```

Comment:
I despise an `eval` in the configuration code. Other than that the PR would be great!

Reply:
So just the following?
```
REMOTE_AUTH_DEFAULT_PERMISSIONS = os.environ.get('REMOTE_AUTH_DEFAULT_PERMISSIONS', '{}')
```
I don't have an idea what would be a good way to build something similar to the way how lists are entered.

([REMOTE_AUTH_DEFAULT_PERMISSIONS](https://github.com/netbox-community/netbox/blob/90dbe9bf60ab3c72be10fd070c65c26dca543ca5/docs/configuration/optional-settings.md#remote_auth_default_permissions) has become [with 2.9](https://github.com/netbox-community/netbox/blob/dde52309d25d6fd6ecc5f05c7a373fa81cb719dc/docs/release-notes/version-2.9.md#configuration-changes) a dictionary.)